cli/directory.go

```diff
@@ -83,7 +85,8 @@ func ValidateDirectory(dir string) error {
 			notFound = append(notFound, f)
 		}
 	}
-	if len(notFound) > 0 {
+	// if neither config.yaml or hasura-config.yaml is found, return error
+	if len(notFound) > 1 {
 		return errors.Errorf("cannot validate directory '%s': [%s] not found", dir, strings.Join(notFound, ", "))
```

Comment:
I'm not sure if this is the best way to validate the project directory, when alternative filenames for configuration file are present. 
If it is, should I rename the error message here to `cannot validate directory | no configuration file found`? As we currently validate a project directory if only a configuration file is found.